launch.sh

```diff
@@ -387,40 +387,42 @@ fi
 cat > /tmp/${vid}.html <<_PAGE_
 <!doctype html>
 <html>
-   <head></head>
-   <body>
-      <style>
-         body {
-         background-color : black;
-         margin : 0;
-         }
-         video {
-         left: 50%;
-         position: absolute;
-         top: 50%;
-         transform: translate(-50%, -50%);
-         width: 100%;
-         max-height: 100%;
-         }
-      </style>
-      <script src="//cdn.jsdelivr.net/npm/hls.js@latest"></script>
-      <video id="video" controls autoplay></video>
-      <script>
-         var video = document.getElementById('video');
-         if(navigator.userAgent.match(/(iPhone|iPod|iPad)/i)) {
-         video.src = '${vid}.m3u8';
-         video.autoplay = true;
-          }
-          else if(Hls.isSupported()) {
-            var hls = new Hls();
-            hls.loadSource('${vid}.m3u8');
-            hls.attachMedia(video);
-            hls.on(Hls.Events.MANIFEST_PARSED,function() {
-              video.play();
-          });
-         }
-      </script>
-   </body>
+<head>
+    <meta charset="utf-8">
+    <title>videojs-ogvjs Demo</title>
+    <link href="https://vjs.zencdn.net/5.11.7/video-js.min.css" rel="stylesheet">
+</head>
+<body>
+<style>
+    .video-js {
+        position: fixed;
+        right: 0;
+        bottom: 0;
+        min-width: 100%;
+        min-height: 100%;
+        width: auto;
+        height: auto;
+        background-color: black;
+        background-size: cover;
+        z-index:99999;
+    }
+</style>
+<video id="videojs-ogvjs-player" class="video-js vjs-default-skin" controls preload="none" width="352" height="264" crossorigin>
+    <source src="${vid}.ogv" type='video/ogg; codecs=&quot;theora, vorbis&quot;'>
```

Comment:
@colleenkhenry you'll need to generate the correct `ogv` files for this- i didn't touch that part.